odoo/tools/translate.py

```diff
@@ -1009,7 +1009,10 @@ def extract_spreadsheet_terms(fileobj, keywords, comment_tags, options):
                 if markdown_link:
                     terms.append(markdown_link[1])
         for figure in sheet['figures']:
-            terms.append(figure['data']['title'])
+            terms.append(figure['data']['title'].get("text", ''))
```

Comment:
json files currently in the code still have the previous version. Both should be supported (and tested) until we have the upgrade scripts working.